app/components/autocomplete/at_mention/index.ts

```diff
@@ -40,6 +41,7 @@ const enhanced = withObservables([], ({database, channelId}: WithDatabaseArgs &
     return {
         useChannelMentions,
         useGroupMentions,
+        localUsers: queryAllUsers(database).observe(),
```

Comment:
What is the difference if we fetch all users and then filter, instead of querying the database and get only the filtered results?

Reply:
Not much. Here some pros and cons of the current approach:

Pros:
- We filter in memory, instead of searching in the database, which my gut feeling tells me that is faster.
- Database queries are where we are used to have them (in the index).
Cons:
- We load more information into memory at once.
- New channels will create a new render when it is not needed.
- Since we are not observing with columns, we may get stale data.

Unless we find the first pro is worth enough, we can create a local query `searchUsers` that receives a term, and gets from the database all users with a name like the term for all relevant fields.

What do you think?